lib/server.coffee

```diff
@@ -126,7 +124,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 
 		Spiderable.originalRequest = req
 		url 		= Spiderable._urlForPhantom Meteor.absoluteUrl(), req.url
-		hash 		= SHA256 url
+		hash 		= hashCode url
```

Comment:
Then why not use another, lighter hash from a standard library?

This package isn't rate limited by the hash on a simple URL. It would be limited by network bandwidth, I'd think... (not by cpu).